example_tinkerbell.py

```diff
@@ -0,0 +1,77 @@
+from fluentm import (
+    Actor,
+    Boundary,
+    Process,
+    DataFlow,
+    HTTP,
+    MTLS,
+    SIGV4,
+    TLS,
+    Unknown,
+    SIGV4,
+    Internal,
+    DHCP,
+    WrappableProtocol,
+)
+from fluentm import report
+
+scenes = {
+    # Example using variables, which is fine for small things but gets hard with longer flows
+    "Tink API traffic": [
+        DataFlow(
+            Actor("tink-cli"),
+            Process("Tink API").inBoundary("Tink Control Plane Network"),
+            MTLS(HTTP("API request")),
+        ),
+        DataFlow(
+            Process("Machine").inBoundary("On Premise Network"),
+            Process("boots").inBoundary("Tink Control Plane Network"),
+            DHCP("IP Request"),
+            # response=DHCP("DHCP Response"),
+        ),
+        DataFlow(
+            Process("boots")
+            .inBoundary("Tink Control Plane Network")
+            .inBoundary("On Premise Network"),
+            Process("Tink API").inBoundary("Tink Control Plane Network"),
+            MTLS(HTTP("API request")),
+            "Boots queries for IP",
+        ),
+        DataFlow(
+            Process("Machine").inBoundary("On Premise Network"),
+            Process("boots").inBoundary("Tink Control Plane Network"),
+            DHCP("PXE Location Request"),
+            # response=DHCP("PXE Location Response"),
+        ),
+        DataFlow(
+            Process("Machine").inBoundary("On Premise Network"),
+            Process("Nginx").inBoundary("Tink Control Plane Network"),
+            HTTP("Boot OS Request"),
+            # response=HTTP("Boot OS"),
+        ),
+        DataFlow(
+            Process("Machine").inBoundary("On Premise Network"),
+            Process("Tink API").inBoundary("Tink Control Plane Network"),
+            TLS(HTTP("Workflow")),
+        ),
+        DataFlow(
+            Process("Machine").inBoundary("On Premise Network"),
+            Process("Container Registry").inBoundary("Tink Control Plane Network"),
+            TLS(HTTP("Container Images")),
+        ),
+        DataFlow(
+            Process("Machine").inBoundary("On Premise Network"),
+            Process("Hegel").inBoundary("Tink Control Plane Network"),
+            TLS(HTTP("Machine metadata")),
+            "Metadata request",
+        ),
+        DataFlow(
+            Process("Machine").inBoundary("On Premise Network"),
+            Process("Nginx").inBoundary("Tink Control Plane Network"),
+            HTTP("Workload OS"),
+        ),
+    ]  # TODO: Fix silent error on repeat label names
```

Comment:
I actually fixed this error but appear to have not removed the comment from all the code yet :P

No need to remove it here, I'll go through and remove this comment everywhere in a separate PR